config/environments/production.rb

```diff
@@ -80,4 +80,5 @@
   config.active_record.dump_schema_after_migration = false
 
   config.action_mailer.delivery_method = :notify
+  config.action_mailer.default_url_options = { host: HostingEnvironment.aytq_domain }
```

Comment:
Assuming we only ever access the support/staff functionality via AYTQ (and not Check), this should be fine. Will need revisiting if we ever start sending emails from Check.

Reply:
Yeah. I wondered about that. At the moment we're only sending invites for staff users I think so it'll be fine for now.